### DISCUSSION TITLE
Point to an existing markdown execution tool.

### DISCUSSION THREAD

**Discussion by joerdav** [pkgxdev/pkgx#294] [Ideas]
I think we've landed on the same conclusion that README.md files are great for documenting scripts and there should be a way to run said scripts.

I've built [https://github.com/joerdav/xc](https://github.com/joerdav/xc) which has seen production experience as a task runner, to great success. I wonder if it's worth signposting while `tea` is still in the state of "coming soon".

**Comment by mxcl** [comment]
This is perfect actually as we're deprecating our concept of executable markdown considering it “out of scope” and instead recommending other tools. SO! I’ll prominently link to xc. Thanks for your work! Refs #292

**Reply by joerdav** [reply]
Brilliant thanks. I'm glad to hear you're removing it because it's "out of scope" not because you realised it was a bad idea! :)

**Reply by mxcl** [reply]
Oh indeed. I think the idea is solid, but we were doing a poor job of it and in such cases it is important IMO to drop it and hope the community picks it up rather than potentially cause the idea to stagnate. I'm super glad to see the idea was already more solidly implemented! Looking forward to trying xc out.

**Reply by mxcl** [reply]
I’ll package up `xc`… unless you beat me to it.

**Reply by joerdav** [reply]
It's already packaged up actually: https://xcfile.dev/getting-started/ 
There's a couple ways to install it.

Unless you mean package up using `tea` I wouldn't know how!

**Reply by mxcl** [reply]
yes, with tea. @mfts made a start https://github.com/teaxyz/pantry.extra/pull/116
